CMakeLists.txt

```diff
@@ -14,7 +14,7 @@
 
 cmake_minimum_required(VERSION 3.18.0)
 
-set(CMAKE_OSX_DEPLOYMENT_TARGET 10.14 CACHE STRING "Minimum OS X deployment version.")
+set(CMAKE_OSX_DEPLOYMENT_TARGET 10.15 CACHE STRING "Minimum OS X deployment version.")
```

Comment:
Is that still needed? You increased macOS requirements when you used `std::filesystem::is_regular_file()`, but you are not anymore.

Reply:
Not for this PR anymore, I have to make another PR for the Julia packaging that does require this though.

Reply:
Should be done for that other PR then.

Reply:
Yes, it should.

Reply:
[8cdb5b8](https://github.com/cellml/libcellml/pull/1157/commits/8cdb5b8d2439138a2b8951ca36e217f2e9287fe9)